execute_until_build_succeeds.py

```
@@ -92,10 +92,10 @@ def execute_until_build_succeeds(
         for attempt in range(max_retries):
             try:
                 test_cases = unit_tests_first
-                if parsed is not None and hasattr(parsed, 'new_unit_test_code'):
-                    test_cases = parsed.new_unit_test_code
+                if parsed is not None and hasattr(parsed, 'test_file_content'):
+                    test_cases = test_file_content
```

Ruff (0.8.0)

Undefined name `test_file_content`

(F821)